### ISSUE TITLE
Student landing page: Saved

### ISSUE THREAD

**Issue by kbwagner** [openopps/openopps-platform#2900]
Who: Student internship
What: Viewing of saved internships
Why: to be able to review saved internships of interest prior to applying

Saved:  This ticket will supplement 2459 once the saved process is in place and this can be fully tested.

The saved section will display
- Viewing x of Y of Z internships
- The table header columns will be Internship status, Internship, and Last updated
- Internship status will display a pill
- There will be a Sort by that will allow to sort by each column heading
- The first 5 saved will display and then a Show more.
- Show more will display all on the page 

Invision Mock: https://opm.invisionapp.com/d/main/#/console/15990896/331677019/preview

Related Tickets: 2459 - Landing page shell ticket 
2899 - testable ticket for viewing application tickets

**Comment by Kibrahacha67** [comment]
@kbwagner Question, in the AC Status is going to be in pill form.  Do we have a set of status "open,
 in progress, closed etc?

**Comment by kbwagner** [comment]
@Kibrahacha67 yes ticket 3000 in the backlog is for the statuses:
Draft - The internship has been saved as a draft but not submitted
Submitted - The internship has been submitted for approval to the community manager
Approved - The internship has been approved by the community manager
Open - the internship has been approved, the application start date for the selected cycle has passed, and the job has not yet been completed
Completed - When the automated closeout process runs in ticket 2915, the internship opportunities for the entire cycle will be set to "Completed"

**Comment by bmcorum** [comment]
@margaretdumas and @Kibrahacha67 this is ready for testing.

**Comment by margaretdumas** [comment]
3/20/19 - Mdumas - testing...

**Comment by kbwagner** [comment]
@raywilliams - returning to development for #1 below and questions for #2 and 3
1) the column headings for saved are fine when I have added saved internships.  But when there are none, they are incorrect:
![image](https://user-images.githubusercontent.com/18709918/54704581-64d94a80-4b11-11e9-9529-f110add537d7.png)
2) I can write this in a new ticket if I need to but when there are no internships saved can we change the wording from existing "no opportunities" to something like "No internship opportunities saved"
3) When there is only one internship saved, it says "Viewing 1-1 of 1 internships".  Is it possible only for when it's 1 to have internship display singular as "Viewing 1-1 of 1 internship".  Anything above 1 would be "internships"?  I know picky!  If not possible all good.  Or, I can put it in a new ticket as well.
![image](https://user-images.githubusercontent.com/18709918/54704649-8d614480-4b11-11e9-91e9-089e09db9ec1.png)

**Comment by raywilliams** [comment]
@kbwagner  I’m fixing the first issue. Please write a new ticket for issues 2 and 3.

**Comment by kbwagner** [comment]
@raywilliams 
Yes, I have written 3218 for 2 and 3 above.

**Comment by bmcorum** [comment]
@margaretdumas and @Kibrahacha67 this is ready for testing.

**Comment by Kibrahacha67** [comment]
3-21-19 LHammond in testing

**Comment by kbwagner** [comment]
3/21/19 - KWagner tested Chrome/Test
- column headings are now correct.
ready for final review.
